src/services/feed_validator.py

```
@@ -82,29 +89,35 @@
                 continue  # Try second attempt
             except aiohttp.ClientError as e:
                 if attempt == 0:
-                    logging.warning(f"First attempt connection error for {url}: {str(e)}")
+                    logging.warning(
+                        f"First attempt connection error for {url}: {str(e)}"
+                    )
                 if attempt < 1:
                     await asyncio.sleep(self.retry_delay)
                 continue  # Try second attempt
             except Exception as e:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)